tests/consensus_tests/test_failed_snapshot_recovery.py

```diff
@@ -159,15 +160,15 @@ def test_failed_snapshot_recovery(tmp_path: pathlib.Path):
             continue
         break
 
-    # Assert storage does not contain initialized flag
+    # Assert storage does not contain initialized flag after restoring on restart
     flag_path = shard_initializing_flag(peer_dirs[-1], COLLECTION_NAME, 0)
     assert not os.path.exists(flag_path)
 
     # Assert that the local shard is dead and empty
     local_shards = get_local_shards(peer_api_uris[-1])
     assert len(local_shards) == 1
     assert local_shards[0]["shard_id"] == 0
-    assert local_shards[0]["state"] == "Partial"
+    assert local_shards[0]["state"] == "Recovery"
     assert local_shards[0]["points_count"] == 0
```

Comment:
Lets assert it's either `Partial` OR `Recovery`. It can be either depending on what shard transfer method is selected. We cannot properly guarantee here that WAL delta is always used.